tests/unit/type/string.tcl

```diff
@@ -816,22 +816,31 @@ if {[string match {*jemalloc*} [s mem_allocator]]} {
 
 if {[string match {*jemalloc*} [s mem_allocator]]} {
     test {Memory usage of embedded string value} {
-        # Check that we can fit 9 bytes of key + value into a 32 byte
+        # Check that we can fit 9 bytes of key + value into a 24 byte
         # allocation, including the serverObject itself.
         r set quux xyzzy
-        assert_lessthan_equal [r memory usage quux] 32
+        assert_lessthan_equal [r memory usage quux] 24
 
         # Check that the SDS overhead of the embedded key and value is 6 bytes
         # (sds5 + sds8). This is the memory layout:
         #
-        # +--------------+--------------+---------------+----------------+
-        # | serverObject |              | sds5 key      | sds8 value     |
-        # | header       | key-hdr-size | hdr "quux" \0 | hdr "xyzzy" \0 |
-        # | 16 bytes     | 1            | 1  + 4    + 1 | 3  + 5     + 1 |
-        # +--------------+--------------+---------------+----------------+
+        # +-------------------+----------+---------------+----------------+
+        # | robj header       | key sds  | sds5 key      | sds8 value     |
```

Comment:
```suggestion
        # | serverObject       | key sds  | sds5 key      | sds8 value     |
```
Is there a reason we reverted the name back to robj?

Reply:
Reverted? In the past it was typedef struct redisObject robj. We de-branded redisObject to serverObject. Robj has bever been changed.

Robj stands for reference-counted object. 😆